packages/e2e-tests/specs/autosave.test.js

```diff
@@ -101,6 +102,43 @@ describe( 'autosave', () => {
 		expect( await getEditedPostContent() ).toEqual( wrapParagraph( 'before save after save' ) );
 	} );
 
+	it( 'should clear sessionStorage upon user logout', async () => {
+		await clickBlockAppender();
+		await page.keyboard.type( 'before save' );
+		await saveDraft();
+
+		// Fake local autosave
+		await page.evaluate( ( postId ) => window.sessionStorage.setItem(
+			`wp-autosave-block-editor-post-${ postId }`,
+			JSON.stringify( { post_title: 'A', content: 'B', excerpt: 'C' } )
+		), await getCurrentPostId() );
+		expect( await page.evaluate( () => window.sessionStorage.length ) ).toBe( 1 );
+
+		const logoutUrl = await page.$eval( '#wp-admin-bar-logout a', ( element ) => element.href );
+		await page.goto( logoutUrl );
```

Comment:
Can't we just click on the button/link?

Reply:
Right, click should work better in this case 👍